platforms/software/dependency-management/src/main/java/org/gradle/api/internal/artifacts/transform/DefaultTransform.java

```diff
@@ -206,8 +206,7 @@ public static void validateInputFileNormalizer(String propertyName, @Nullable Fi
                         .forProperty(propertyName)
                         .label("is declared to be sensitive to absolute paths")
                         .documentedAt(userManual("validation_problems", "cacheable_transform_cant_use_absolute_sensitivity"))
-                        .noLocation()
-                        .category(VALIDATION, "property", TextUtil.screamingSnakeToKebabCase(CACHEABLE_TRANSFORM_CANT_USE_ABSOLUTE_SENSITIVITY))
+                        .category(CATEGORY_VALIDATION, "property", TextUtil.screamingSnakeToKebabCase(CACHEABLE_TRANSFORM_CANT_USE_ABSOLUTE_SENSITIVITY))
```

Comment:
🤔 If these constants are meant to be supplied to a `.category()` method, then I don't think having their names begin with `CATEGORY_` is necessary, or clearer.

Reply:
Fixed, please take a look.